### PR TITLE
Modal: Added scroll shadows

### LINKED ISSUE #3060: [Innspill til komponent]: Modal. Indikasjon på at det er mer innhold når container blir scrollable
### Tilbakemelding

F.eks et pil-ikon eller en blur-effekt som som aktiveres når det er innhold under eller over viewport, som forsvinner om man har scrollet helt til toppen eller til bunnen.

### PR DESCRIPTION
**Pull request by HalvorHaugan**
Closes https://github.com/navikt/aksel/issues/3060

### CODE REVIEW

File: @navikt/core/css/modal.css [hunk 1]
```diff
@@ -128,6 +128,18 @@
   overflow: auto;
   overscroll-behavior: contain;
   position: relative; /* Needed to make sr-only elements position correctly - see Storybook */
+  background:
+    linear-gradient(white 30%, transparent) top,
+    linear-gradient(transparent, white 60%) bottom,
```
Comment (KenAJoh) [inline @navikt/core/css/modal.css]: Would it be possible to use `inherit` here instead of `white` in case the user changed the background?

**Review by KenAJoh — COMMENTED** [review]
Love the implementation with ` background-attachment: local, local, scroll, scroll;` 🤩

File: @navikt/core/css/modal.css (hunk 1 above)
Comment (HalvorHaugan) [reply @navikt/core/css/modal.css]: `inherit` doesn't work.

Comment (KenAJoh) [reply @navikt/core/css/modal.css]: Not optimal, but works with tokens atleast:
```
.navds-modal {
  --__ac-modal-bg: var(--ac-modal-bg, var(--a-surface-default));

  background-color: var(--__ac-modal-bg);
  border: none;
  border-radius: var(--a-border-radius-large);
  box-shadow: var(--a-shadow-xlarge);
  padding: 0;
  position: fixed;
  max-height: 100%;
  max-width: 100%;
  color: var(--a-text-default);
}

.navds-modal__body {
  padding: var(--a-spacing-6);
  overflow: auto;
  overscroll-behavior: contain;
  position: relative; /* Needed to make sr-only elements position correctly - see Storybook */
  background:
    linear-gradient(var(--__ac-modal-bg) 30%, transparent) top,
    linear-gradient(transparent, var(--__ac-modal-bg) 60%) bottom,
    radial-gradient(farthest-side at 50% 0, rgba(0 0 0 / 0.3), transparent) top,
    radial-gradient(farthest-side at 50% 100%, rgba(0 0 0 / 0.3), transparent) bottom;
  background-size:
    100% 40px,
    100% 50px,
    100% 16px,
    100% 16px;
  background-attachment: local, local, scroll, scroll;
  background-repeat: no-repeat;
}
```

**Comment by HalvorHaugan** [comment]
> Love the implementation with ` background-attachment: local, local, scroll, scroll;` 🤩

Not my idea, but quite elegant 😄

File: @navikt/core/css/modal.css (hunk 1 above)
Comment (HalvorHaugan) [reply @navikt/core/css/modal.css]: Nice! I totally forgot that we had a token for bg color 😂

**Review by KenAJoh — APPROVED** [review]

**Pull request merged**